pyproject.toml

```diff
@@ -116,7 +116,6 @@ optional = [
 ]
 test = [
     'asv',
-    'codecov',
```

Comment:
I initially forgot to run `tools/generate_pyproject.toml.py` and was surprised to find that our pre-commit hook doesn't catch this case. I think it's because the file filter is too restrictive.

https://github.com/scikit-image/scikit-image/blob/1a2e55891c20cb70dfa04a4dbc731a287ad957af/.pre-commit-config.yaml#L37-L43

Reply:
I think it should trigger on changes to `requirements/*.txt`, `pyproject.toml`, `tools/*pyproject*`. Not sure yet how to update this field accordingly. [pre-commits docs](https://pre-commit.com/#hooks-files) just say 
> (optional: default '') the pattern of files to run on.

Reply:
It's a regular expression, so you can combine files with `|`, i.e. something like:

```
pyproject.toml|requirements/.*\.txt|tools/.*pyproject.*
```

Reply:
Fixed up.